data/schema/units/abilities.cfg

```diff
@@ -41,6 +41,19 @@
 	{FILTER_TAG "filter_base_value" base_value ()}
 	{WML_MERGE_KEYS}
 [/tag]
+[tag]
+	name="halo"
+	max=infinite
+	super="units/unit_type/abilities/~generic~"
+	{SIMPLE_KEY image string}
```

Comment:
I'd prefer to call it `halo_image`.

I'm wondering if it's better to have this need to be in a `[halo]` tag, or whether it should be support in any ability regardless of the tag name. Requiring the tag to be called `[halo]` seems good for getting it into 1.18 quickly. Calling it `halo_image` would allow us to use `[halo]halo_image=` in 1.18, and then easily decide to allow `[any_tagname]halo_image=` in 1.19.

Reply:
changed to halo_image

Reply:
I meant rename the `image=` attribute, not the `[halo]` tag.

Reply:
done